docs/hardware.rst

```diff
@@ -13,9 +13,43 @@ Hardware Overview
 
 For an installation of SecureDrop, you must acquire:
 
-.. include:: includes/pre-install-hardware.txt
+.. include:: includes/pre-install-hardware-required.txt
 
-In the sections that follow, we provide additional details on each item.
+Additionally, you may want to consider the following purchases:
+
+.. include:: includes/pre-install-hardware-optional.txt
+
+In the sections that follow, we provide additional details on most of these
+items.
+
+.. tip::
+
+    While a printer is not required, we highly recommend it. Printing documents
+    is generally far safer than copying them in digital form. See our
+    :doc:`journalist guide <journalist>` for more information.
```

Comment:
The journalist guide is large. Let's add a section link to the "Working with Documents" within the Journalist Guide to aid the reader a bit here.

Reply:
Done.